apps/full-site-editing/package.json

```diff
@@ -133,9 +133,9 @@
 	},
 	"devDependencies": {
 		"@types/wordpress__plugins": "^2.3.6",
-		"@babel/preset-typescript": "^7.10.1",
-		"@types/jest": "^26.0.3",
+		"@babel/preset-typescript": "^7.8.3",
+		"@types/jest": "^25.2.1",
 		"@wordpress/eslint-plugin": "^6.0.0",
-		"babel-jest": "^26.1.0"
+		"babel-jest": "^26.0.1"
```

Comment:
Thanks for the update. Guess it's best to use `lerna add` to add packages as I assume it'd automatically align versions?